themes/geekboot/layouts/partials/docs-sidebar.html

```diff
@@ -1,67 +1,41 @@
 {{ $current := . }}
-{{ $version_section := (where .Site.Sections ".Page.Params.version" .Page.Params.version) }}
+
 <nav class="bd-links-nav w-100" aria-label="Docs navigation">
+  {{ with .Site.GetPage "section" .Section }}
+    {{ $sectionPages := (.Pages | append .) }}
+    {{ range $sectionPages.ByWeight }}
+      {{ $expand := (eq $current .) }}
+      {{ $expand = (and (eq $current.CurrentSection .) (ne .FirstSection .)) }}
+      {{ $id := substr (sha1 .Permalink) 0 8 }}
 
-  <div class="bd-links-container">
-    <div class="container nav-container pe-0 d-flex w-100">
-    <div class="container flex-row pe-0">
-      <div class="d-flex flex-column ">
-            <div class="d-flex"><a href="{{ .FirstSection.Permalink }} ">{{ .FirstSection.Title }}</a></div>
-      </div>
-    </div>
-  </div>
-    {{ range ((index $version_section 0).Pages).GroupBy "Weight" }}
-        {{ range .Pages.ByWeight }}
-          {{ if not .Params.tocHidden }}
-            {{ template "sidebar-nav" dict "childPages" .Pages "current" $current "thisPage" .}}
+      {{ if not .Page.Params.tocHidden }}
+        <div class="section-container container pe-0 pt-1">
+          <div class="container nav-container pe-0 d-flex w-100 {{if eq . $current}}active{{ else if $expand }}active-parent{{end}}">
+            <div class="d-flex w-100 border-0"><a href="{{.Permalink}}">{{ .Title }} </a></div>
```

Comment:
![CleanShot 2023-01-24 at 15 15 13@2x](https://user-images.githubusercontent.com/492942/214307026-61302324-5058-490f-8c6e-405cc72e67df.png)

Reply:
This is a Thing™.

Can you open an issue for this one and I can work on it in the near future?

Reply:
Sure will do